### PR TITLE
Package coq-wasm.0.1

### PR DESCRIPTION
**Pull request by raoxiaojia**
Mechanised WebAssembly 1.0 Specification in Coq, with an extracted OCaml interpreter

### CODE REVIEW

File: released/packages/coq-wasm/coq-wasm.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
```
Comment (palmskog) [inline released/packages/coq-wasm/coq-wasm.0.1/opam:36]:
```suggestion
```
The tests are failing the build.

File: released/packages/coq-wasm/coq-wasm.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  ]
+]
+install: [
+[make "install"]
+]
```
Comment (palmskog) [inline released/packages/coq-wasm/coq-wasm.0.1/opam:41]:
```suggestion
```
The `Makefile` just has a single action `default` that calls `opam` again. That won't work, let's have Dune handle the install...

File: released/packages/coq-wasm/coq-wasm.0.1/opam [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+dev-repo: "git+https://github.com/WasmCert/WasmCert-Coq.git"
+url {
+  src: "https://github.com/WasmCert/WasmCert-Coq/archive/refs/tags/0.1.tar.gz"
+  checksum: "sha256=8b128a9a8f3acaa5ded5f05c12f07fe5828e40d961bc00fedf53a252a67f5890"
+}
+tags: [
+  "keyword: WebAssembly"
+  "category: Computer Science/Semantics and Compilation/Semantics"
+  "date: 2023-10-24"
```
Comment (palmskog) [inline released/packages/coq-wasm/coq-wasm.0.1/opam:50]:
```suggestion
  "keyword:WebAssembly"
  "category:Computer Science/Semantics and Compilation/Semantics"
  "date:2023-10-24"
```

**Review by palmskog — COMMENTED** [review]
The build fails, I think this will fix it.

File: released/packages/coq-wasm/coq-wasm.0.1/opam (hunk 2 above)
Comment (raoxiaojia) [reply released/packages/coq-wasm/coq-wasm.0.1/opam]: I see -- this was my first PR on the coq repo and I was blindly looking at some previous PRs to submit this one. Thanks a lot for the corrections!

**Pull request merged**